app/src/layouts/DashboardLayout/MenuHeader/ProductsDropDown.tsx

```diff
@@ -88,23 +88,33 @@ const Products: React.FC<ProductsProps> = (props) => {
 
   const products: ProductProps[] = [
     {
-      title: "Web Debugger",
+      title: "HTTP Rules (Web Debugger)",
       icon: RulesImg,
-      description: "Lightweight web debugging proxy to modify HTTPs request & response.",
+      description: "Intercept and modify requests, inject scripts, Map Local, Map Remote, etc.",
       handleClick: useCallback(() => {
         trackProductClickedInDropDown("web_debugger");
         props.toggleDropDown();
         redirectToRules(navigate);
       }, [navigate, props]),
     },
```

Comment:
No need to wrap handleClick functions with `useCallback`, instead wrap `products` in useMemo